conftest.py

```diff
@@ -57,3 +58,27 @@ def small_client(small_cluster):
         client.wait_for_workers(10)
         client.restart()
         yield client
+
+
+@pytest.fixture(scope="session")
+def s3_region():
+    return "us-east-2"
+
+
+@pytest.fixture(scope="session")
+def s3_bucket():
+    return "dask-io"
```

Comment:
Once we move the other S3 path-related fixtures here, we can probably make this a module-level `S3_BUCKET` variable too as I don't think we'll be using an feature features anymore

Reply:
I've made an `S3_BUCKET` variable.